Test/JsonDiffTest.cs

```diff
@@ -66,22 +66,28 @@ public static IEnumerable<object[]> ProviderForNumericDiffTest()
                                      { 'b': 10 },
                                      { 'c': 20 },
                                     ]
-                                 }", 
+                                 }",
                                @"{
                                'a': {
                                     '_t': 'a',
                                      '0': {
                                            'b': {
                                                    'left': 15,
                                                    'right': 10,
-                                                    'diff': 5
+                                                    'Int Diff': {
+                                                      'Subtraction': 5,
+                                                      'Division': 1
+                                                      }
                                                 }
                                                 },
                                     '1': {
                                           'c': {
                                                    'left': 25,
                                                    'right': 20,
-                                                    'diff': 5
+                                                    'Int Diff': {
```

Comment:
To vary the tests as much as possible, reverse the order of the 20 and the 25 and confirm that the subtraction diff comes out at -1. I think the division would stay the same